utils/run_validation.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import matplotlib as mpl
+mpl.rcParams['text.usetex'] = False
+import matplotlib.pyplot as plt
+from desc.twinkles import validate_ic
+
+if __name__ == "__main__":
+
+    cat_folder = sys.argv[1]
```

Comment:
How is this script meant to be run?  If it is supposed to be run from the command line, can you use `argparse` to get the arguments so that `python utils/run_validation.py -h` gives some helpful information?